config/php-cs-fixer.php

```diff
@@ -41,6 +41,10 @@
 
             // function notation
             'native_function_invocation' => ['include' => ['@all']],
+            'nullable_type_declaration' => [
+                'syntax' => 'question_mark',
+            ],
```

Comment:
This seems to be the [default value, at least in the latest version](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/language_construct/nullable_type_declaration.rst), so we don't need to explicitly specify it.  Also, it would belong in the `language_construct` section.

Reply:
The default for `syntax` is `question_mark`, so I think we could just have `'nullable_type_declaration' => true` (based on my renewed understanding of how the config system works).  But maybe we'd like to explicitly state the preference anyway - IDK.

It's definitely in the wrong section though, and should be in `language_construct`.

Reply:
Done and repushed.